frontend/src/components/eMIB/Email.jsx

```diff
@@ -22,6 +22,7 @@ const styles = {
   email: {
     textAlign: "left",
     paddingLeft: 16,
+    paddingRight: 16,
     paddingTop: 16
```

Comment:
Can we simply put:

> padding: 16

So there will be a padding also at the bottom, which is also a good thing and also less code for that style.